web/src/pages/api/public/media/[mediaId].ts

```diff
@@ -0,0 +1,123 @@
+import { z } from "zod";
+
+import { env } from "@/src/env.mjs";
+import { getMediaStorageServiceClient } from "@/src/features/media/server/getMediaStorageClient";
+import {
+  GetMediaQuerySchema,
+  GetMediaResponseSchema,
+  PatchMediaBodySchema,
+} from "@/src/features/media/validation";
+import { createAuthedAPIRoute } from "@/src/features/public-api/server/createAuthedAPIRoute";
+import { withMiddlewares } from "@/src/features/public-api/server/withMiddlewares";
+import {
+  ForbiddenError,
+  InternalServerError,
+  LangfuseNotFoundError,
+} from "@langfuse/shared";
+import { Prisma, prisma } from "@langfuse/shared/src/db";
+import { recordIncrement, recordHistogram } from "@langfuse/shared/src/server";
+
+export default withMiddlewares({
+  GET: createAuthedAPIRoute({
+    name: "Get Media data",
+    querySchema: GetMediaQuerySchema,
+    responseSchema: GetMediaResponseSchema,
```

Comment:
do we need custom rate limits here?

Reply:
I assume this is an add on.

Reply:
Can be added later if necessary, is there a good reason to add them already now?

Reply:
add limits now

Reply:
will be added in https://linear.app/langfuse/issue/LFE-2682/add-rate-limits-to-media-create-patch